### PR TITLE
post: TeraSaturday 7

### PR DESCRIPTION
**Pull request by skaldarnar**

### CODE REVIEW

File: _posts/2020-12-05-terasaturday-7.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+_Welcome to the return of TeraSaturday, a not-so-weekly update post about all the neat stuff that has been going on in
+the world of Terasology! Half blog, half changelog, completely cool._
+
+## 📰 Since last TeraSaturday ...
+
+Another week has passed, and we just released [Terasology Alpha 17 (Release Candidate 2)](https://github.com/MovingBlocks/Terasology/releases/tag/v4.1.0-rc.2) with a bunch of performance bug fixes and performance improvements. We updated the TerasologyLauncher ([latest release v4.2.0](https://github.com/MovingBlocks/TerasologyLauncher/releases/tag/v4.2.0)) so that you can also start the latest release candidates on your Mac 🍏.
+
+_@DarkWeird_ tracked down some zombie threads that stopped the game from exiting properly, _@Michael P_ tweaked the lighting code a bit to remove the overexposure in bright daylight, and _@Skaldarnar_ fixed the issue where killing mobs in Josharia's Survival would crash the server.
+
+Our ... distinctive spawn point features (🙄) have been smoothed out by _@4D enthusiast_ who also continued to erase the remnants of the `SurfaceHeightFacets` from all modules.
```
Comment (Cervator) [inline _posts/2020-12-05-terasaturday-7.md]:
```suggestion
Our ... distinctive spawn point features (🙄) have been smoothed out by _@4D enthusiast_ who also continued to erase the remnants of the deprecated `SurfaceHeightFacets` from all modules.
```

Trivial clarification - one might wonder what poor said facet has been doing to be erased. Although maybe that's just wordy 🤷

File: _posts/2020-12-05-terasaturday-7.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Furthermore, new contributors _@Paul_ and _@Gwhoyanne_ fixed the time synchronization between host and clients and now want to add some permissions to our magic time travel keys (<kbd>F3</kbd> to enable debug mode, then do time travel via <kbd>→</kbd> or <kbd>←</kbd>).
+
+In total, that's [21 engine RPs](https://github.com/search?q=org%3AMovingBlocks+type%3Apr+merged%3A2020-11-28..2020-12-05) and [40 PRs on modules](https://github.com/search?q=org%3ATerasology+type%3Apr+merged%3A2020-11-28..2020-12-05) since November 28. 
+
+![World Generation Improvements]({{ site.baseurl }}/img/terasaturday/7/worldgen.jpg)
+
+## 🚀 Module/Feature Highlights
+
+Since we focussed on stabilization and bug fixes this week not much happened regarding gameplay features. _@Niruandaleth_ and _@Skaldarnar_ did a pass over the in-game help present in Josharia's Survival and worked on some usability aspects.
```
Comment (Cervator) [inline _posts/2020-12-05-terasaturday-7.md]: This _does_ seem like a major improvement! Don't really have a specific suggestion here, but it feels like your achievement is being minimized by saying not much overall changed. Maybe something like  "Not much happened _other_ than this one cool usability thing"

**Review by Cervator — DISMISSED** [review]
Nothing much to say, pretty good as-is 👍

**Comment by skaldarnar** [comment]
Thanks for the review, both points have been addressed.

**Pull request merged**